server/app.js

```diff
@@ -37,3 +53,29 @@ app.get("/proxy/:proxyUrl*", (req, res) => {
 app.listen(port, () => {
   console.log("Server started on port 3000!");
 });
+
+app.get("/repos", async (_, res) => {
+  const response = await octokit.paginate("GET /orgs/uclaacm/repos", {
+    per_page: 100,
+  });
+
+  const result = new Map();
+  result.set("no-topic", []);
+
+  for (let obj of response) {
+    if (!!obj.homepage) {
```

Comment:
in javascript, `obj.homepage` will be a false-y value if it doesn't exist and truthy if the homepage property exists, so the double `!!` is not necessary!

Reply:
yup i'll delete that real quick.